source/NVDAObjects/IAccessible/winConsole.py

```diff
@@ -4,9 +4,19 @@
 #See the file COPYING for more details.
 #Copyright (C) 2007-2019 NV Access Limited, Bill Dengler
 
+import config
+
+from winVersion import isWin10
+
 from . import IAccessible
 from ..window.winConsole import WinConsole
 
 class WinConsole(WinConsole, IAccessible):
 	"The legacy console implementation for situations where UIA isn't supported."
-	pass
\ No newline at end of file
+	pass
+
+def findExtraOverlayClasses(obj, clsList):
+	if isWin10(1703) and config.conf['terminals']['keyboardSupportInLegacy']:
+		from NVDAObjects.behaviors import TerminalKeyboardSupport
+		clsList.append(TerminalKeyboardSupport)
+	clsList.append(WinConsole)
```

Comment:
I think this might be a bit ambiguous. is this NVDAObjects.IAccessible.winConsole.WinConsole or NVDAObjects.window.winConsole.WinConsole. I'm sure it will be the former, but a linter might complain that you're importing WinConsole and then override it. Please change the import above to something like:
``from ..window.winConsole import WinConsole as WinConsoleWindow`